controllers/component_build_controller_pac.go

```diff
@@ -902,3 +1153,61 @@ func retrievePipelineSpec(bundleUri, pipelineName string) (*tektonapi.PipelineSp
 	pipelineSpec := pipelineSpecObj.PipelineSpec()
 	return &pipelineSpec, nil
 }
+
+func updateIncoming(repository *pacv1alpha1.Repository, incomingSecretName string, pacIncomingSecretKey string, targetBranch string) bool {
```

Comment:
Please add a description to the function.

Reply:
ok